pkg/manifests/configfile.go

```diff
@@ -121,7 +121,7 @@ func (cf *ConfigFile) RelativeConfigPath() (string, error) {
 // necessary) according to the config file.
 func (cf *ConfigFile) GenerateManifests(ctx context.Context, manifests Manifests) ([]byte, error) {
 	if cf.PatchUpdated != nil {
-		finalBytes, _, _, err := cf.getGeneratedAndPatchedManifests(ctx, manifests)
+		_, finalBytes, _, err := cf.getGeneratedAndPatchedManifests(ctx, manifests)
```

Comment:
Ouch

Reply:
shit happens! I didn't catch it during the review either